src/test/scala/mesosphere/marathon/core/launchqueue/impl/RateLimiterActorTest.scala

```diff
@@ -34,25 +35,26 @@ class RateLimiterActorTest extends AkkaUnitTest {
   "RateLimiterActor" should {
     "GetDelay gets current delay" in new Fixture {
       rateLimiter.addDelay(app)
-
-      val delay = (limiterRef ? RateLimiterActor.GetDelay(app.configRef)).futureValue.asInstanceOf[RateLimiter.DelayUpdate]
-      assert(delay.delayUntil == Some(clock.now() + backoff))
+      val delayUpdate = (limiterRef ? RateLimiterActor.GetDelay(app.configRef)).futureValue.asInstanceOf[RateLimiter.DelayUpdate]
+      assert(delayUpdate.delay.map(_.deadline).contains(clock.now() + backoff))
```

Comment:
I don't think this is quite right since the assertion would be true for now delay update as well. Scalatest has a nice matcher for this

```suggestion
      delayUpdate.delay.value.deadline should be(clock.now() + backoff)
```